components/UI/Filters/components/Filter/Filter.module.css

```diff
@@ -1,19 +1,25 @@
 .filter {
     position: relative;
     overflow: hidden;
-    margin: 16px 16px 0;
+    padding: 0 16px;
+    opacity: 0;
     box-sizing: content-box;
+    visibility: hidden;
```

Comment:
I think because of this filter content appears smoothly but disappears instantly. Can we fix it?

Reply:
ah, I take it back. it's probably because of dummy-component. it works alright with real filters